pysat/tests/test_meta.py

```diff
@@ -819,25 +884,25 @@ def test_inequality_with_higher_order_meta3(self):
         assert not (self.meta == meta3)
 
     def test_assign_higher_order_meta(self):
+        """Test assign higher order metadata.
+        """
         meta = pysat.Meta()
         meta['dm'] = {'units': 'hey', 'long_name': 'boo'}
         meta['rpa'] = {'units': 'crazy', 'long_name': 'boo_whoo'}
         self.meta['higher'] = meta
 
     def test_assign_higher_order_meta_from_dict(self):
-        meta = pysat.Meta()
-        meta['dm'] = {'units': 'hey', 'long_name': 'boo'}
-        meta['rpa'] = {'units': 'crazy', 'long_name': 'boo_whoo'}
-        self.meta['higher'] = {'meta': meta}
-
-    def test_assign_higher_order_meta_from_dict_correct(self):
```

Comment:
these two tests were identical, but only one had assert statements

Reply:
I wish I could tell you what I was thinking when I wrote that one. A docstring probably would've helped. 🤷

Reply:
Let's remove the one without assert statements.